### ISSUE TITLE
IANA Reservation for Header

### ISSUE THREAD

**Issue by alficles** [cta-wave/common-media-client-data#25]
The specification says:

> The header has a reserved case-insensitive field name:
> Common-Media-Client-Data

If we're going to reserve an HTTP header, we should probably go through the process of registering it with IANA. I'm not certain our process here qualifies for a Permanent Header, but I believe it qualifies for a Provisional. Hopefully, we can find an expert that can help with the process here, or we can determine that we don't wish to register it, in which case we should probably drop the word "reserved" from the specification.

**Comment by wilaw** [comment]
Thomas Edwards provided more information:

Here is the IANA registry of “message headers” (which could be used with HTTP):
 
https://www.iana.org/assignments/message-headers/message-headers.xml
 
There are “Permanent Message Header Field Names” defined by IETF RFCs and “Provisional Message Header Field Names” defined by other sources (e.g. W3C).  IANA registration is described in RFC 3864 “Registration Procedures for Message Header Fields”.
 
The mail address for announcement of new header field submissions is ietf-message-headers@lists.ietf.org
 
Info on the list is here:
 
https://www.ietf.org/mailman/listinfo/ietf-message-headers
 
The archives linked on that page go to 404, but the messages appear to be here:
 
https://ietf.org/mail-archive/text/ietf-message-headers/

**Comment by wilaw** [comment]
Here is an example of the info we need to provide. We may need to have the spec finished and available online before we can register it:

Header field name: traceparent
Applicable protocol: http
Status: permanent
Change Controller:
   Distributed Tracing Working Group
   https://www.w3.org/2018/distributed-tracing/
   public-trace-context@w3.org
Specification:
   https://www.w3.org/TR/trace-context/#traceparent-field

**Comment by alficles** [comment]
Requirements for Permanent registration:

```
   o  A Permanent Message Header Field Registry, intended for headers
      defined in IETF standards-track documents, those that have
      achieved a comparable level of community review, or are generally
      recognized to be in widespread use.  The assignment policy for
      such registration is "Specification Required", as defined by RFC
      2434 [3], where the specification must be published in an RFC
      (standards-track, experimental, informational or historic), or as
      an "Open Standard" in the sense of RFC 2026, section 7 [1].
```

Since this isn't an RFC, it needs to meet the requirements of an "Open Standard":

```
(1)  Open Standards

      Various national and international standards bodies, such as ANSI,
      ISO, IEEE, and ITU-T, develop a variety of protocol and service
      specifications that are similar to Technical Specifications
      defined here.  National and international groups also publish
      "implementors' agreements" that are analogous to Applicability
      Statements, capturing a body of implementation-specific detail
      concerned with the practical application of their standards.  All
      of these are considered to be "open external standards" for the
      purposes of the Internet Standards Process.
```

I think the standards produced here should qualify.

**Comment by wilaw** [comment]
Agreed we should wait until CMCD spec is published and then go back and immediately register the header with IANA.

**Comment by wilaw** [comment]
Just updating this thread to indicate that we now have to register our 4 headers, instead of a single header.

CMCD-Request
CMCD-Object
CMCD-Status
CMCD-Session

**Comment by wilaw** [comment]
Template that I propose to submit to IETF per https://tools.ietf.org/html/rfc3864 

**Header field name:**
Four headers are requested under this submission, all defined within the same specification:

CMCD-Request
CMCD-Object
CMCD-Status
CMCD-Session

**Applicable protocol:**
http (RFC [2616](https://tools.ietf.org/html/rfc2616))

**Status:**
Standard. "CTA-5004 - Common Media Client Data"

**Author/Change controller:**
Consumer Technology Association (CTA)
1919 S. Eads Street
Arlington, VA 22202
https://www.cta.tech/
standards@cta.tech
703-907-7600

**Specification document(s):**
      https://shop.cta.tech/collections/standards/products/web-application-video-ecosystem-common-media-client-data-cta-5004 

**Related information:**
 None.

**Comment by wilaw** [comment]
Status? @wilaw

**Comment by ablasgen** [comment]
@wilaw I'm definitely happy to help, just need instructions! (Sorry if we didn't respond earlier, the message might have gotten buried or filtered.)

**Comment by wilaw** [comment]
Thanks @ablasgen ! No worries. Let's see what @alficles suggests before we engage your support.

**Comment by wilaw** [comment]
IANA have officially approved our application, so this issue can be closed as resolved. 


> From: Amanda Baber via RT <[iana-mime@iana.org](mailto:iana-mime@iana.org)>
> Sent: Monday, May 11, 2026 8:50 PM
> To: Alexandra Blasgen <[ABlasgen@cta.tech](mailto:ABlasgen@cta.tech)>
> Subject: [IANA #1450200] application/cmcd registration request
> 
> Dear Alexandra,
> 
> We've registered the following media type:
> 
> application/cmcd
> 
> Please see
> 
> https://urldefense.com/v3/__http://www.iana.org/assignments/media-types__;!!GjvTz_vk!SH5rdDYQXhE-IA6velQ0lQ8DI7eFC-1n-ameOjjCcGxk9HqdEnsP3eV2h2vkyaD6p1wxQaACI-N_eE4$
> 
> If you need to update the registration or its contact information, you can reach us at this address or at [iana@iana.org](mailto:iana@iana.org).
> 
> Best regards,
> 
> Amanda Baber
